util.py

```diff
@@ -498,7 +498,7 @@ def guess_rmantree_initial():
     rmantree = rmantree_from_env()
     if rmantree != '':
         vstr = rmantree.split('-')[-1]
-        vf = float(vstr.strip('/'))
```

Comment:
Haha yes I had the same problem.  Yes to that.

On Tue, May 17, 2016 at 8:05 AM, Jon D <notifications@github.com> wrote:

> In util.py
> <https://github.com/bsavery/PRMan-for-Blender/pull/260#discussion_r63539589>
> :
>
> > @@ -498,7 +498,7 @@ def guess_rmantree_initial():
> >      rmantree = rmantree_from_env()
> >      if rmantree != '':
> >          vstr = rmantree.split('-')[-1]
> > -        vf = float(vstr.strip('/'))
>
> Could be. Problem is that first \ would cause Python to ignore the
> quotation after it so shouldn't it be / followed by two forward slashes? I
> can't even write it down right here cause the comment box erases one of the
> forward slashes.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/bsavery/PRMan-for-Blender/pull/260/files/b8e3c49db63603647c5355305041e14938b18623#r63539589>
>



-- 
brian.savery@gmail.com
508-274-8700